appveyor.yml

```diff
@@ -50,7 +50,7 @@ build_script:
   # install openapi-generator locally
   - mvn clean install --quiet -Dorg.slf4j.simpleLogger.defaultLogLevel=error
   # run the locally installed openapi-generator-gradle-plugin
-  - gradle -b modules\openapi-generator-gradle-plugin\samples\local-spec\build.gradle buildGoSdk --info
+  - gradle -b modules\openapi-generator-gradle-plugin\samples\local-spec\build.gradle buildGoSdk --stacktrace
```

Comment:
I'd like to leave `--stacktrace` here rather than `--info`. It was taking ~2 hours to get the feedback from AppVeyor after commit and I was unable to repro the issue in Windows locally.